staging/src/k8s.io/apiserver/pkg/storage/storagebackend/config.go

```diff
@@ -83,5 +85,6 @@ func NewDefaultConfig(prefix string, codec runtime.Codec) *Config {
 		Codec:                codec,
 		CompactionInterval:   DefaultCompactInterval,
 		DBMetricPollInterval: DefaultDBMetricPollInterval,
+		HealthcheckTimeout:   2 * time.Second,
```

Comment:
Following similar pattern of existing code, could we define a `DefaultHealthcheckTimeout`. I think it improves code readability.

Reply:
@jingyih done :)